frontend/www/js/omegaup/components/problem/Edit.vue

```diff
@@ -201,6 +201,7 @@
 
       <div class="tab-pane active" v-if="showTab === 'tags'">
         <omegaup-problem-tags
+          v-bind:errors="errors"
```

Comment:
la variable `errors` nunca se modifica. es necesario pasarla y declararla en este componente? no sería mejor

```suggestion
          v-bind:errors="[]"
```

o hacer que el prop `errors` en `omegaup-problem-tags` sea `[]` por default? (creo que prefiero el segundo)

Reply:
Ahh, se me olvidó quitarlo. Creo que por el momento no es necesario tenerlo en ninguno de los dos componentes